text/0000-cbool.md

```diff
@@ -0,0 +1,28 @@
+- Feature Name: cbool
+- Start Date: Mon Mar  9 00:16:53 CET 2015
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+Specify that `bool` is compatible with the `_Bool` C type.
+
+# Motivation
+
+You cannot safely call ffi functions with boolean arguments without a compatible type.
+
+# Detailed design
+
+Specify that `bool` is compatible with the `_Bool` C type and inform the compiler that `bool` is ffi safe.
+
+# Drawbacks
+
+None right now. This should be a no-op on all supported platforms.
```

Comment:
>In any case, the precise representation of C types is usually chosen by the compiler, not the hardware.

That's wrong. The representation of _Bool is part of the ABI and (for all hardware developed after C99) defined by the hardware vendor. E.g. 

http://www.x86-64.org/documentation/abi.pdf
http://infocenter.arm.com/help/topic/com.arm.doc.ihi0042e/IHI0042E_aapcs.pdf